fastlane/Fastfile

```diff
@@ -1077,14 +1077,16 @@ lane :check_pods do
     verbose: true,
     podspec: 'RevenueCat.podspec',
     platforms: 'ios,osx,tvos,watchos,visionos',
-    fail_fast: true
+    fail_fast: true,
+    allow_warnings: true
   )
   pod_lib_lint(
```

Comment:
in what circumstances do we actually use `pod_lib_lint` with Xcode 16? 
Asking because this change does worry me a bit - the whole point of pod lib lint is to find warnings that we otherwise miss. It might be more valuable to keep this running in Xcode 15 if it can still warn us about stuff that isn't this, or otherwise when running in Xcode 16 maybe update the concurrency mode

Reply:
I see your point above removing the warnings making things less safe, and I have to agree with you! We currently run `pod_lib_lint` in both the `pod-lib-lint` and `create-tag` (via the release-checks lane) workflows. I think it's probably fine to continue using Xcode 15 here - I'll try it out and see how it goes!

I did double check, and unfortunately `pod_lib_lint` provides no way to only ignore certain warnings 😅

Reply:
@aboedo I've updated the PR so that the `pod-lib-lint` and `create-tag` workflows use Xcode 15!